pyne/alara.py

```diff
@@ -142,7 +149,7 @@ def photon_source_to_hdf5(filename, nucs='all', chunkshape=(10000,)):
     f.seek(0)
     G = len(header) - 2
 
-    dt = np.dtype([
+    dtype = np.dtype([
```

Comment:
Do you think we should use a different name: `response_dtype` or something?  Just to avoid confusion with the built in name `dtype`